types/is-secret/tslint.json

```
@@ -0,0 +1,6 @@
+{ 
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "export-just-namespace": false
+    }
```

```suggestion
    "extends": "dtslint/dt.json"
```
If you accept the top level exports, apply this, otherwise skip